components/shared/profile/edit/EditProfile.tsx

```diff
@@ -0,0 +1,451 @@
+            >
+              Remove
+            </Button>
+          </div>
+        </>
+      ))}
+      <CustomButton
+        buttonType="profileButton"
+        type="button"
+        // @ts-ignore
+        onClick={() => appendGoal({ name: "", completed: false })}
+      >
+        Add Goal
+      </CustomButton>
+
+      {experienceFields.map((field, index) => {
+        return (
+          <>
+            <div className="profile-goals-wrapper" key={field.id}>
+              <Button
+                className="text-white-100"
+                type="button"
+                onClick={() => removeExperience(index)}
+              >
+                {" "}
+                Remove
+              </Button>
+              <Input
+                id={`experiences[${index}].name`}
+                type="text"
+                {...register(`experiences[${index}].name` as any)}
+              />
+            </div>
+          </>
+        );
+      })}
+      <CustomButton
+        buttonType="profileButton"
+        type="button"
+        onClick={() => appendExperience({ name: "" })}
+      >
+        Add Experience
+      </CustomButton>
+      {/* Technologies Section with Search Box */}
+      <div className="flex flex-col justify-stretch gap-2 px-3.5 py-3  ">
+        <label
+          htmlFor="technologies"
+          className="justify-start space-y-2 text-white-300"
+        >
+          Tech Stacks
+        </label>
+        <div className="flex w-full  bg-black-700 ">
+          {technologies && // eslint-disable-next-line array-callback-return
+            technologies?.map((tech: any, index) => {
+              const icon = techStackBadges.find((badge) => badge.name === tech);
+              if (icon)
+                return (
+                  <>
+                    <span
+                      key={index}
+                      className="tech-badges paragraph-3-medium shadow-custom w-1/2   flex-row justify-between bg-black-600 text-white-100"
+                    >
+                      {icon.icon({ size: 16 })}
+                      {tech}
+                    </span>
+                  </>
+                );
+            })}
+
+          <Input
+            className=" w-1/2  bg-black-700 text-white-100"
+            value={search}
+            onChange={(e) => setSearch(e.target.value)}
+          />
+        </div>
+        <div className="flex h-9 w-full flex-row">
+          {search &&
+            results.length > 0 &&
+            // eslint-disable-next-line array-callback-return
+            results.map((result: any, index) => {
+              if (technologies?.includes(result.name)) {
+                return false;
+              }
+              const icon = techStackBadges.find(
+                (badge) => badge.name === result.name
+              );
+              if (icon)
+                return (
+                  <>
+                    <div className="flex  flex-col bg-black-700" key={index}>
```

Comment:
```suggestion
        <div className="flex w-full flex-row flex-wrap bg-black-700 py-2">
          {search &&
            results.length > 0 &&
            // eslint-disable-next-line array-callback-return
            results.map((result: any, index) => {
              if (technologies?.includes(result.name)) {
                return false;
              }
              const icon = techStackBadges.find(
                (badge) => badge.name === result.name
              );
              if (icon)
              return (
                <>
                  <div className="flex flex-col" key={index}>
```

I don't think each button needs the background. I think the container for the results does. Remove the hard height on the container. Add a flex wrap. Give it some padding. It looks much better.

![image](https://github.com/agordon123/git_note/assets/4108484/20fc5ea8-ffd5-48dd-8cd9-ebd606f1dec9)